Makefile

```diff
@@ -53,9 +53,15 @@ install:
 
 TEMP_FILE = temp_parser_file
 
-parser:
-	go get github.com/pingcap/goyacc
-	go get github.com/qiuyesuifeng/golex
+golex:
+	$(GO) get github.com/qiuyesuifeng/golex
```

Comment:
we'll get rid of golex tool sooner: https://github.com/pingcap/tidb/issues/1490 ,
put golex into vendor increase lot of files, so maybe it's cleaner this way.